src/crimson/common/logclient.cc

```diff
@@ -352,12 +341,23 @@ seastar::future<> LogClient::handle_log_ack(Ref<MLogAck> m)
     const LogEntry &entry(*q);
     if (entry.seq > last)
       break;
-    ldout(cct,10) << " logged " << entry << dendl;
+    logger().debug(" logged {}", entry);
     q = log_queue.erase(q);
   }
   return seastar::now();
 }
 
+LogChannelRef LogClient::create_channel(const std::string& name) {
+  LogChannelRef c;
+  if (channels.count(name))
+    c = channels[name];
```

Comment:
this is an anti pattern when using an associated map in STL. instead of looking up the channel twice, i'd suggest something like
```c++
auto it = channels.find(name);
if (it == channels.end()) {
  it = channels.insert(it, {name, seastar::make_lw_shared<LogChannel>(this, name)})
}
return it->second;
```